### PR TITLE
solution

### PR DESCRIPTION
**Pull request by OleksandrNi**

### CODE REVIEW

**Comment by KostiaO** [comment]
Хорошее решение, но можно было упростить...

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Check all points from checklist

**Comment by OleksandrNi** [comment]
add variables

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -29,6 +29,17 @@
...
   // write code here
+  let sum = days * 40;
+
+  if (days > 6) {
+    sum = sum - 50;
+  } else {
+    if (days > 2) {
+      sum = sum - 20;
+    }
+  }
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:40]: You should create variables for 20 and 50 numbers, now I can't understand their meaning
And check point 3 in checklist (good example of returning value from if statement)

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -29,6 +29,19 @@
...
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days > 6) {
+    return sum - bigDiscount;
+  } else {
+    if (days > 2) {
+      return sum - smallDiscount;
+    }
+  }
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:42]: else is redundant here (point 3 in checklist)
```suggestion
  if (days > 6) {
    return sum - bigDiscount;
  }
  
  if (days > 2) {
    return sum - smallDiscount;
  }
  
```

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**